### ISSUE TITLE
Inline chat with sticky scroll looses top area

### ISSUE THREAD

**Issue by bpasero** [microsoft/vscode#219542]
Steps to Reproduce:

1. bring up inline chat with a lengthy response
2. make sure sticky scroll is enabled
3. notice how when inline chat widget grows up, the top part moves out of sight

![Image](https://github.com/user-attachments/assets/b4b11421-3218-4308-a5a6-ec96969cc5ac)

![Image](https://github.com/user-attachments/assets/b1a3affe-941e-4413-8f99-a86353c4086a)

//cc @aiday-mar

**Comment by bpasero** [comment]
This can happen also when initially triggering inline chat:

![Image](https://github.com/user-attachments/assets/dd6ab68c-c5c5-47b9-9611-913fc6830104)

@jrieken as a mitigation I wonder why we not simply reveal the inline chat in center of the view port when it opens?

**Comment by jrieken** [comment]
> @jrieken as a mitigation I wonder why we not simply reveal the inline chat in center of the view port when it opens?

Yeah, there two problems. Revealing enough lines on start (when sticky scroll is on) and continuous revealing and resizing the inline chat scrolls upwards.

**Comment by jrieken** [comment]
With https://github.com/microsoft/vscode/pull/234974 we have the min-bar as suggested above (https://github.com/microsoft/vscode/issues/219542#issuecomment-2408448450)

**Comment by bpasero** [comment]
This is probably not an issue anymore with V2
